package.json

```diff
@@ -27,18 +27,18 @@
     "chalk": "^2.4.1",
     "cosmiconfig": "^5.0.0",
     "create-jest-runner": "^0.5.3",
-    "eslint": "^4.0.0 || ^5.0.0 || ^6.0.0"
+    "eslint": "^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.1.0",
     "@babel/core": "^7.1.0",
     "@babel/preset-env": "~7.7.0",
     "babel-jest": "^25.1.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-config-prettier": "^4.3.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jest": "^22.6.4",
-    "eslint-plugin-prettier": "^3.1.0",
+    "eslint-config-airbnb-base": "^14.1.0",
```

Comment:
I _think_ this is why CI fails - keep it at 13?

Reply:
No; the airbnb config does not yet support eslint 7. Thus, this PR is blocked until https://github.com/airbnb/javascript/issues/2223 is resolved.